docs/flow/tutorials/multiple-inputs.md

```diff
@@ -0,0 +1,76 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/multiple-inputs
+title: Multiple Inputs
```

Comment:
Ditto the last comment: the content here is fine, but I think this could benefit from a Prometheus-specific title like "Chaining Prometheus metrics filtering" or something like that.